### DISCUSSION TITLE
Improving browser-scaling of "Birdseye" Streams

### DISCUSSION THREAD

**Discussion by cmmh** [blakeblackshear/frigate#2824] [Ask A Question]
Wondering if there are browser settings for firefox for chrome that would improve the scaling of the birdeye videos.  All of my cameras are 1080p or better, but the resulting video in the birdseye view looks super blocky, like the images were just decimated rather than having any real scaling applied to them.  

Perhaps the solution is to adjust the birdseye configuration?

**Comment by blakeblackshear** [comment]
The resolution of the Birdseye stream is set in the configuration file.
